src/scenes/Reports/AcceleratorReportView.tsx

```diff
@@ -101,7 +114,9 @@ const AcceleratorReportView = () => {
           size='medium'
         />
         <Button
-          disabled={selectedReport?.status !== 'Not Submitted' && selectedReport?.status !== 'Needs Update'}
+          disabled={
+            (selectedReport?.status !== 'Not Submitted' && selectedReport?.status !== 'Needs Update') || !targetsAreSet
+          }
```

Comment:
We might want to create a list of statuses somewhere for updateable components in this view, since the next PR will also have several. Could do something like this?
```ts
const UPDATABLE_STATUSES = ['Not Submitted', 'Needs Update']
...
disabled={UPDATABLE_STATUSES.includes(selectedReport?.status}
```

This isn't required for approval or anything, just commenting on it.

Reply:
I think that's a good optimization we should probably make. Although it's possible the disabled logic changes after feedback from QA, so I think handling it in the next PR makes sense.